core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -165,6 +165,12 @@ public class SchemaRegistryConfig extends RestConfig {
   public static final String COMPATIBILITY_CONFIG = "avro.compatibility.level";
   public static final String SCHEMA_COMPATIBILITY_CONFIG = "schema.compatibility.level";
 
+  /**
+   * <code>schema.cache.maximum.weight</code>
+   */
+  public static final String SCHEMA_CACHE_MAXIMUM_WEIGHT_CONFIG = "schema.cache.maximum.weight";
+  public static final int SCHEMA_CACHE_MAXIMUM_WEIGHT_DEFAULT = 1000000;
```

Comment:
The sum of canonical schema length must be under 1,000,000 characters is what this variable means. From the guava cache docs there's a lot of fuzziness as to how things get invalidated within the cache itself (based on usage and % of contribution to weight total?) but we've set this limit such that large schemas with many references should be getting invalidated over time. What this essentially means is that more smaller schemas will be cached (and that's good!) as we should be able to cache quite a few of them. No significant performance changes seem to occur because of this change and would love guidance for testing